lib/chitragupta/util.js

```diff
@@ -63,6 +63,12 @@ function populateWorkerData(dataParam) {
   data.meta.format.version = formatVersions.WORKER;
 }
 
+function populateDefaultFormatData(dataParam) {
+  const data = dataParam;
+  data.meta.format.category = categories.SERVER;
+  data.meta.format.version = formatVersions.SERVER;
```

Comment:
Won't you need to change the version in `format_versions.js`

Reply:
No, not requried. Just want to use the existing versions as the default. The current versions are already at 5.